src/sconsx/config.py

```diff
@@ -371,7 +372,7 @@ def ALEASolution(options, tools=[], dir=[]):
     if isinstance(platform, Win32):
         # Checking for compiler info first
         compileroptions = deepcopy(options)
-        compilerconf = Config(default_tools,dir)
+        compilerconf = Config([],dir)
```

Comment:
This may have a huge effect, isn't it?
How users that depend on compiler will add the compiler? Explicitly?

Reply:
Just explain a bit. Thanks

Reply:
default_tools is already included by default in Config construction.